src/utils/stringifyENSNFTAvatar.ts

```diff
@@ -0,0 +1,11 @@
+export default function stringifyENSNFTAvatar({
```

Comment:
could you add / explain the pattern for this NFTs resolution as a comment for the future? also for parseENSNFTAvatar. I  think there was also a pattern that ENS follows to allow a PFP to be NFTs, is this it?